MC-42708: Scheduled Export does not triggered by cron (#1418)
magento/merchdocs: src/system/data-schedule-import.md

    The new scheduled import job is added to the list on the _Scheduled Import/Export_ page. From this page it can be run immediately for testing and edited. The import file is validated before the execution of each import job.

{:.bs-callout-info}

> Scheduled import makes changes to Magento system configuration. After saving, make sure you follow the cache invalidation notice that will appear on top of the Magento admin page and flush the cache in order to apply the new or updated schedule.

Reviewer: ```suggestion
Scheduled import makes changes to the system configuration. After saving, make sure you address the cache invalidation notice that appears at the top of the Admin page and flush the cache in order to apply the new or updated schedule.
```
